submit.go

```diff
@@ -192,7 +193,7 @@ func (s *submitServer) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	}
 
 	// Filter out unwanted rageshakes, if a list is defined
-	if len(s.cfg.AllowedAppNames) != 0 && !s.cfg.allowedAppNameMap[p.AppName] {
+	if !s.allowedAppNameMap[p.AppName] {
```

Comment:
this means we will not accept any rageshakes if the list is unconfigured (in contrast to what the printed warning and documentation say)?

Reply:
You're right - fixed, and patch now contains a (simple) test that triggers the three cases by creating a submitServer with no config.